### ISSUE TITLE
csv-writing does not provides `make-csv-printing-params`

### ISSUE THREAD

**Issue by alinarezrangel** [jbclements/csv-writing#2]
Hello! Today I found that when trying to configure `display-table`, csv-writing does not provides `make-csv-printing-params`.

Example:

```racket#lang racket
(require csv-writing)
(display-table '((1 2 3) (4 5 6))
               #:printing-params (make-csv-printing-params
                                  #:quotes-only-when-needed? #f))
```

The fix was: in [line 7 of `main.rkt`](https://github.com/jbclements/csv-writing/blob/2b6ac969c42113893559ddfe88bcfeafa1f4b221/csv-writing/main.rkt#L7) change:

```racket
 table-cell->string)
```

To:

```racket
 table-cell->string
 make-csv-printing-params)
```

Thanks for the library!

**Comment by jbclements** [comment]
ouch! I've moved the relevant tests into a separate file, to catch this and other similar errors (there were a few). 

Closed by #4710b3c75efb0d9e

**Comment by jbclements** [comment]
Also: thanks!
